### ISSUE TITLE
Can't sort on createdAt field

### ISSUE THREAD

**Issue by aref00** [strapi/strapi#2417]
**Informations**
- **Node.js version**:  v11.2.0
- **NPM version**:  6.4.1
- **Strapi version**:3.0.0-alpha.14.5
- **Database**: mongo
- **Operating system**: ubuntu

**What is the current behavior?**
I get >ERROR Error: "GraphQL error: Unkown field: "createdAt""

I'm using Angular 7 for front-end

**Steps to reproduce the problem**
dependencies for graphql queries, that I'm using in angular:
    apollo-angular
    graphql-tag

my query:

> 
```
{
        reports( sort: "createdAt:DESC", limit: 20, start: ${page * 20}) {
          _id
          title
          short
          image{
            url
          }
          creator {
            username
          }
          createdAt
          categories{
            name
          }
        }
      }
```

I'm using strapi in production. 

**What is the expected behavior?**
Getting data object

**Suggested solutions**
?

**Comment by lauriejim** [comment]
Hello @aref00 , thank you for reporting this issue. We are currently on another feature and bug fix. Since time is lacking on our side, feel free investigate and submit a PR, we’ll appreciate your contribution on this issue! 

Check out the contributing guide to get started: https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md

**Comment by kamalbennani** [comment]
Probably related to: https://github.com/strapi/strapi/pull/2419

**Comment by derrickmehaffy** [comment]
Unable to reproduce on Strapi v19 on bookshelf

**Comment by derrickmehaffy** [comment]
Marking as closed as I believe this was fixed in #2419
